src/partials/welcome.ts

```diff
@@ -301,24 +317,43 @@ class WelcomeController {
         const keyStore = new saltyrtcClient.KeyStore(decrypted.ownSecretKey);
 
         // Set up the broadcast channel that checks whether we're already connected in another tab
-        this.setupBroadcastChannel(keyStore.publicKeyHex);
-
-        // Reconnect
-        this.reconnect(keyStore, decrypted);
+        this.setupBroadcastChannel(keyStore.publicKeyHex, WelcomeController.BROADCAST_DELAY)
+            .then((result) => {
+                this.$scope.$apply(() => {
+                    switch (result) {
+                        case 'already_open':
+                            this.log.warn('Session already connected in another tab or window');
+                            break;
+                        case 'no_answer':
+                            this.log.debug('No broadcast received indicating that a session is already open');
```

Comment:
We log this line here, but not in the "start" case. Do we need it at all, or was it mainly for debugging?

Reply:
Because in the `scan` case, there's no delay. That's the main difference.